ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/checkboxesActivityPicklistQuestion.component.ts

```diff
@@ -33,7 +33,7 @@ import { NGXTranslateService } from '../../../services/internationalization/ngxT
                           [checked]="getOptionSelection(option.stableId)"
                           [disabled]="readonly"
                           [disableRipple]="true"
-                          (change)="select($event.checked, option.stableId, option.exclusive)">
+                          (change)="select($event.checked, option.stableId, option.exclusive); option.allowDetails ? updateCharactersLeftIndicator(option.stableId) : null">
```

Comment:
This line already looks a little long.
I think this could be refactored to just called `optionChanged(checked: boolean, option: ActivityPicklistOption)`